app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/AttachmentOptions.kt

```diff
@@ -64,7 +63,7 @@ fun AttachmentOptionsComponent(
     isFileSharingEnabled: Boolean,
     modifier: Modifier = Modifier
 ) {
-    Box(modifier.height(KeyboardHeight.DEFAULT_KEYBOARD_TOP_SCREEN_OFFSET)) {
+    Box(modifier = Modifier.height(KeyboardHeight.DEFAULT_KEYBOARD_TOP_SCREEN_OFFSET)) {
```

Comment:
remove `modifier: Modifier` from the `AttachmentOptionsComponent`s signature if you don't use it. 
Otherwise it may be confusing: "why do I provide modifier to AttachmentOptionsComponent and it's not uplied"